tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -631,6 +631,23 @@ def standard_deviation(x):
     return np.std(x)
 
 
+@set_property("fctype", "simple")
+def variation_coefficient(x):
+    """
+    Returns the variation coefficient of x
```

Comment:
Could you include a short sentence on what the variant coefficient is (e.g. std / mean)?